### ISSUE TITLE
[Flaky Test] [sig-node] Summary API [NodeConformance] when querying /stats/summary should report resource usage through the stats api

### ISSUE THREAD

**Issue by thejoycekung** [kubernetes/kubernetes#95017]
**Which jobs are flaking**:
`node-kubelet-master` (`ci-kubernetes-node-kubelet`)

**Which test(s) are flaking**:
`Summary API [NodeConformance] when querying /stats/summary should report resource usage through the stats api`

**Testgrid link**:
https://testgrid.k8s.io/sig-release-master-blocking#node-kubelet-master

**Reason for failure**:
```
_output/local/go/src/k8s.io/kubernetes/test/e2e_node/summary_test.go:54
Failed after 15.045s.
Expected
    <string>: Summary
to match fields: {
.Node.SystemContainers[pods].CPU:
	Expected
	    <string>: CPUStats
	to match fields: {
	.UsageNanoCores:
		Expected
		    <uint64>: 0
		to be >=
		    <int>: 10000
	}
	
}
```

**Anything else we need to know**:
https://storage.googleapis.com/k8s-gubernator/triage/index.html?test=when%20querying%20%2Fstats%2Fsummary%20should%20report%20resource%20usage%20through%20the%20stats%20api
* 5 clusters of 67 failures (7 in last day)

So far noticed as flaking in these two runs:
* https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-node-kubelet/1308821020399898624
* https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-node-kubelet/1308774969034936320

/sig node

**Comment by RobertKielty** [comment]
Additional notes

This failure spans multiple tests that seem to be targeting different images 

The error message seems to indicate that there is issue contacting the Boskos allocated resources


More instances of the test producing non-deterministic results on

https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-node-kubelet/1308774969034936320
https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-node-kubelet/1307470301163950080
https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-node-kubelet/1307685215744823296
https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-node-kubelet/1308851667558469632
https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-node-kubelet/1308821020399898624

The error logs consistently include the following : 

>   ERROR: (gcloud.compute.addresses.describe) Could not fetch resource:
The resource 'projects/k8s-infra-e2e-boskos-087/regions/us-west1/addresses/bootstrap-e2e-master-ip' was not found

**Comment by alejandrox1** [comment]
Taking a look
Will assign myself a primary point of contact
/assign @alejandrox1 

As for communication, please share this issue in slack or ping
/cc @kubernetes/sig-node-test-failures 
otherwise hard to track down where we can help :smile:

**Comment by alejandrox1** [comment]
and actually, communication may be better over mailing lists - it's super easy to lose messages in slack and prow already sends email notifications.
At least for node, we have the main mailing list and we also have this one https://groups.google.com/g/kubernetes-sig-node-test-failures just for CI-related things.

**Comment by thejoycekung** [comment]
Looks like mirrored by #95343 and #95344

Possibly fixed by #95345 ?

**Comment by alejandrox1** [comment]
Let's move to https://github.com/kubernetes/kubernetes/issues/95344
Since it seems that @harche already working on this and the other issues have more detail :rocket: 
/close
